### DISCUSSION TITLE
How do I disable connection verification?

### DISCUSSION THREAD

**Discussion by ghost** [bromite/bromite#1010] [Q&A]
Can I turn off the window at the top that checks the Internet connection?
![Screenshot_20210318-085921_Trebuchet](https://user-images.githubusercontent.com/79337849/111575772-0221f480-87a7-11eb-864a-c209f5074836.png)

**Comment by chromer030** [comment]
It's really annoying sometimes, @csagan5 is it possible to implement a flag to disable it please ?

**Reply by ghost** [reply]
Same but it ain't worth it having a flag to disable it. Maintaining patches is cumbersome that's why Carl doesn't accept feature requests unrelated to privacy

**Reply by chromer030** [reply]
It's annoying when using browser on Internal networks and Intranets.
